helm/chart/maesh/values.yaml

```diff
@@ -69,7 +69,6 @@ metrics:
     enabled: true
 
 smi:
-  deploycrds: true
```

Comment:
Why this change?

Reply:
Because CRD deployment is handled within the helm command in v3, not via values.